atomium/templates/node/node.tpl.php

```diff
@@ -35,19 +35,13 @@
 
   <?php if ($display_submitted): ?>
       <footer>
-        <?php print $user_picture; ?>
```

Comment:
The rest of the PR is great but: why are we removing this lines (this specific one and the ones at the bottom) from the node.tpl? I think I'm missing some extra information to understand this change.

Reply:
Hi,

Most use cases doesn't need to print the user picture inside the node template, this is why it has been removed.

Reply:
I still find it weird that we remove functionality without it being asked :( Anyways I'm not going to block the issue but I'd like to have a second opinion on this, so I'll keep the review open but move the ticket forward

Reply:
As you wish bruh'

Reply:
I agree with @imanoleguskiza - is weird to rmove functionnality, furthermore in ec_resp we have the picture. So for me, we need to keep the picture.

Reply:
Ok, cancelling this change now.